src/app/group/group.module.ts

```diff
@@ -3,15 +3,21 @@ import { CommonModule } from '@angular/common';
 import { GroupListComponent } from './group-list/group-list.component';
 import { AddEditGroupComponent } from './add-edit-group/add-edit-group.component';
 
+import { RouterModule,Routes } from '@angular/router';
 
+const groupRoutes : Routes = [
+  {path:'gList', component :GroupListComponent},
+  {path:'gAddEdit', component :AddEditGroupComponent}
+]
```

Comment:
Use better naming conventions for all route path as I mentioned in other comments